src/BlazorUI/Bit.BlazorUI/Components/DatePicker/BitDatePicker.razor.cs

```diff
@@ -688,75 +724,75 @@ private int GetValueForComparison(int firstDay)
         return firstDay > firstDayOfWeek ? firstDayOfWeek : firstDayOfWeek - 7;
     }
 
-    private bool CheckMonthForMaxAndMinDate(ChangeDirection direction)
+    private bool IsMonthOutOfMinAndMaxDate(ChangeDirection direction)
```

Comment:
it's better to rename this method to something like `CanChangeMonth` based on its content.
the same goes for other similar methods too.